schema/encoding/jsonschema/benchmark_test.go

```diff
@@ -42,11 +42,13 @@ func BenchmarkEncoder(b *testing.B) {
 			Schema: studentSchema,
 		},
 	}
+	buf := new(bytes.Buffer)
 	for i := range testCases {
 		tc := testCases[i]
 		b.Run(tc.Name, func(b *testing.B) {
 			for i := 0; i < b.N; i++ {
-				enc := jsonschema.NewEncoder(new(bytes.Buffer))
+				buf.Truncate(0)
```

Comment:
Sorry but this is a strawman argument. we completely control the test loop. The encoder is only using the io.Writer interface.